get/opensearch.go

```diff
@@ -0,0 +1,121 @@
+package get
+
+import (
+	"context"
+	"fmt"
+
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/internal/format"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type openSearchCmd struct {
+	resourceCmd
+	PrintPassword bool `help:"Print the password of the OpenSearch BasicAuth User. Requires name to be set." xor:"print"`
+	PrintUser     bool `help:"Print the name of the OpenSearch BasicAuth User. Requires name to be set." xor:"print"`
+	PrintPort     bool `help:"Print the port of the OpenSearch instance. Requires name to be set." xor:"print"`
+}
+
+func (cmd *openSearchCmd) Run(ctx context.Context, client *api.Client, get *Cmd) error {
+	return get.listPrint(ctx, client, cmd, api.MatchName(cmd.Name))
+}
+
+func (cmd *openSearchCmd) list() client.ObjectList {
+	return &storage.OpenSearchList{}
+}
+
+func (cmd *openSearchCmd) print(ctx context.Context, client *api.Client, list client.ObjectList, out *output) error {
+	openSearchList, ok := list.(*storage.OpenSearchList)
+	if !ok {
+		return fmt.Errorf("expected %T, got %T", &storage.OpenSearchList{}, list)
+	}
+	if len(openSearchList.Items) == 0 {
+		return out.printEmptyMessage(storage.OpenSearchKind, client.Project)
+	}
+
+	if cmd.Name != "" && cmd.PrintPassword {
+		return cmd.printPassword(ctx, client, &openSearchList.Items[0], out)
+	}
+	if cmd.Name != "" && cmd.PrintUser {
+		fmt.Fprintln(out.writer, storage.OpenSearchUser)
+		return nil
+	}
+	if cmd.Name != "" && cmd.PrintPort {
```

Comment:
Is this necessary? Because we use ingress, I think this is not even true. So I'd just remove it.